### PR TITLE
Spxis/kodomo support

### PR DESCRIPTION
**Pull request by spxis**
Added new organizations, images, provinces/terrorities, remaining Kodomo mappings and Move Lists for all Kodomo techniques. One note: Kodomo techniques have been prefixed with a string which would allow support for a grouping feature in the Judging GUI. It can be stripped out before displaying but useful for grouping.

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -75,9 +100,104 @@ const KNK_MOVE_LIST = ['Opening Ceremony', 'Kesa-gatame', 'Kata-gatame', 'Kami-s
 const JNK_MOVE_LIST = ['Opening Ceremony', 'Tsuki-dashi', 'Kata-oshi', 'Ryote-dori', 'Kata-mawashi', 'Ago-oshi', 'Kiri-oroshi', 'Ryokata-oshi', 'Nanami-uchi', 'Katate-dori', 'Katate-age', 'Obi-tori', 'Mune-oshi', 'Tsuki-age', 'Uchi-oroshi', 'Ryogan-tsuki', 'Closing Ceremony'];
```
Comment (spxis) [inline src/utils.js:100]: NOTE:  I ultimately think all these move lists should come from a JSON file, perhaps one per kata.

File: src/utils.js [hunk 2]
```diff
@@ -1,30 +1,51 @@
+
 export const ORG_MAP = {
```
Comment (spxis) [inline src/utils.js:2]: NOTE: I'd like to propose using 3-4 chars for these IDs... not that there are going to be that many more provinces, etc... however having two had made it easier to have collisions... and having more letters might make it a little more guessible... again, not an issue as I can't see much changes, but typically ISO would be 3 chars minimum for most states/provinces, and countries as well... which is the 2/3

File: package.json [hunk 3]
```diff
@@ -25,7 +25,7 @@
     "postcss": "^8.4.31",
     "unstorage": "github:apparentvisuals/unstorage#dist",
     "uuid": "^9.0.1",
-    "vue": "^3.2.47",
+    "vue": "^3.3.0",
```
Comment (spxis) [inline package.json:28]: Fixes an issue where I couldn't run the local application.

File: src/utils.js (hunk 2 above)
Comment (apparentvisuals) [reply src/utils.js:2]: provinces/territories are all 2 letters, I shouldn't have bothered using JO for Judo Ontario, should've just used ON. Too late now without migrating all the old values, but as you say, it's unlikely the number of organizations will grow

File: src/utils.js (hunk 1 above)
Comment (apparentvisuals) [reply src/utils.js:100]: 👍🏼

Comment (spxis) [reply src/utils.js:100]: I can do this as I will know instantly if it worked or not...

File: src/utils.js (hunk 2 above)
Comment (spxis) [reply src/utils.js:2]: I'll take your lead on how you want them changed. Just use the 2 character official provincial code?

Comment (apparentvisuals) [reply src/utils.js:2]: yea, let's go with that, should be enough for us.

Comment (spxis) [reply src/utils.js:2]: I've converted all province two-letter codes to official ones from Canada.

File: src/utils.js [hunk 4]
```diff
@@ -75,9 +100,104 @@ const KNK_MOVE_LIST = ['Opening Ceremony', 'Kesa-gatame', 'Kata-gatame', 'Kami-s
...
-const KO5_MOVE_LIST = ['Opening Ceremony', 'Ushiro-ukemi', 'Yoko-ukemi', 'Mae-mawari-ukemi', 'Kumi-kata & Ugoki-kata', 'Happo-no-kuzushi', 'De-ashi-harai', 'Uki-otoshi', 'Uki-goshi', 'Closing Ceremony'];
-const KO6_MOVE_LIST = ['Opening Ceremony', 'Ushiro-ukemi Sit x2', 'Ushiro-ukemi Grip x2', 'Yoko-ukemi 1/Side', 'Mae-mawari-ukemi', 'Mae-mawari-sabaki', 'Ushiro-mawari-sabaki', 'Ayumi-ashi', 'Tsugi-ashi Side', 'Tsugi-ashi Circle', 'Hiza-guruma', 'Tai-otoshi', 'Closing Ceremony'];
-const KO7_MOVE_LIST = ['Opening Ceremony', 'Ushiro-ukemi Back x2', 'Ushiro-ukemi Sit x2', 'Yoko-ukemi 1/Side', 'Outen-ukemi 1/Side', 'Ma-sabaki', 'Ushiro-sabaki', 'Ayumi-ashi', 'Tsugi-ashi', 'Mae-sabaki Throwing', 'Ushiro-sabaki Throwing', 'Closing Ceremony'];
+
+// KODOMO NO KATA lists
+// Optional Numeric prefixing used to Group List Items.
+// A simple regex check can be done to rip out prefix before displaying item.
+// GUI can detect changes between prefix to determine/indicate new group.
+const KO1_MOVE_LIST = [
```
[the remark this answers is not in the document]
Comment (spxis) [reply src/utils.js:108]: Good idea yes, I can go through all the katas and group.

[the remark this answers is not in the document]
Comment (spxis) [reply src/utils.js:106]: Added groups for the Kodomo techniques.

[the remark this answers is not in the document]
Comment (spxis) [reply src/utils.js:106]: For now, I've left it as a lower-case group-id - which would still require us to write a conversion for each group id to a display name... if you don't want this, then I can make it a display name.

[the remark this answers is not in the document]
Comment (spxis) [reply src/utils.js:222]: All katas have been done.

**Comment by spxis** [comment]
So most changes related to Kodomo and Groupings are now done.

**Pull request merged**